jack-sys/src/lib.rs

```diff
@@ -4,20 +4,30 @@ use lazy_static::lazy_static;
 mod consts;
 mod types;
 
+// Contains both static and dynamic dispatched functions.
 mod functions {
     include!(concat!(env!("OUT_DIR"), "/functions.rs"));
 }
 
 pub use consts::*;
-pub use functions::*;
+pub use functions::dynamic_linking;
+pub use functions::dynamic_loading;
```

Comment:
Shouldn't this be gated behind the feature? I think you now also need to have libjack available at compile time even when using the dynamic loading feature.